extensions/jackson/runtime/src/main/java/io/quarkus/jackson/runtime/ObjectMapperProducer.java

```diff
@@ -63,6 +63,9 @@ public ObjectMapper objectMapper(@All List<ObjectMapperCustomizer> customizers,
         for (ObjectMapperCustomizer customizer : sortedCustomizers) {
             customizer.customize(objectMapper);
         }
+
+        objectMapper.getFactory().setBufferRecyclerPool(HybridJacksonPool.INSTANCE);
```

Comment:
What if the user decides to use their own ObjectMapper, as defined in https://quarkus.io/guides/resteasy#json, and forgets to set the correct buffer recycler pool. May be raise a warning? Or force the buffer recycler pool in any other way?

Reply:
That's a very good point, we need to see how it behaves

Reply:
indeed, good point!